clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -456,6 +477,7 @@ message Query {
         PrefixQuery prefixQuery = 22;
         ConstantScoreQuery constantScoreQuery = 23;
         GeoPolygonQuery geoPolygonQuery = 24;
+        SpanNearQuery spanNearQuery = 25;
```

Comment:
Instead of adding an entry for each span query type, would it make sense to add a single entry for `SpanQuery`?

Reply:
Yeah, that makes sense. Although this way all of span queries should be nested under `spanQuery`, but I think it is easier to maintain.